### ISSUE TITLE
Add multiple palettes to the Ink font

### ISSUE THREAD

**Issue by khaledhosny** [aliftype/aref-ruqaa#35]
Blocked on https://forum.glyphsapp.com/t/allow-color-palette-indices-in-color-layer-colors/20937

**Comment by davelab6** [comment]
In GF we use this sample string,

`الحب سماء لا تمطر غير الأحلام`

and we see only red glyphs:

<img width="1388" alt="Screen Shot 2022-03-04 at 4 23 21 PM" src="https://user-images.githubusercontent.com/261579/156855316-0429771f-17b9-4b4e-b21e-7121b7dccd72.png">

but the sample string in your README image (`﴿الحُبُّ سَمَاءٌ لَا تُمطرُ غَيرَ الأَحلَامِ﴾`) is showing red and blue, as in your image:

<img width="1375" alt="Screen Shot 2022-03-04 at 4 23 57 PM" src="https://user-images.githubusercontent.com/261579/156855318-80895965-7dbd-47e1-9994-d04571da1902.png">

Is this working as expected, or is there more work to do on the pallettes?

**Comment by khaledhosny** [comment]
Yes, this is working as expected. This issue is about adding different palette(s) so instead of red and blue, one gets, say, black and red. May be also add a dark theme palette with colors tuned to work better with dark background (though the current palette seems to be doing OK, so to be investigated further).

**Comment by khaledhosny** [comment]
The font currently sets letters red, marks blue and the ornate parentheses gets both red and blue. An alternate palette would keep the same setup, but use different colors.
